code/__HELPERS/icons.dm

```diff
@@ -915,6 +919,12 @@ var/global/list/friendly_animal_types = list()
 		final_average = BlendRGB(final_average, colour, 1)
 	return final_average
 
+/proc/IsBlankIcon(icon/I)

```

Comment:
better idea, generate a static ```/icon``` with no appearance data, and compare if the two appearances are the same.
if this ```/icon``` is blank, they should share an appearance